contracts/JBTiered721DelegateStore.sol

```diff
@@ -483,7 +587,10 @@ contract JBTiered721DelegateStore is IJBTiered721DelegateStore {
 
         while (_currentSortIndex != 0) {
```

Comment:
Shouldn't we exclude an identical active tier already existing (eg
```
if(_tierToAdd.contributionFloor ==
            _storedTierOf[msg.sender][_currentSortIndex].contributionFloor) revert DUPLICATE_FLOOR();
```)

Reply:
no problem if multiple tiers have the same floor